docs/usage/security-and-permissions.md

```diff
@@ -0,0 +1,56 @@
+# Security and Permissions
+
```

Comment:
Comment to remind us that we need to discuss our security stance (when to make a security disclosure, etc), and add it to this document.